plugins/slo/pkg/slo/metrics.go

```diff
@@ -134,7 +143,74 @@ func (p *PrometheusQueryImpl) IsHistogram() bool {
 	return p.isHistogram
 }
 
-// @Note: Assumption is that JobID is valid
+func selectBestMatch(metrics []string) string {
+	min := int(^uint(0) >> 1) // largest int
```

Comment:
You can use `math.MaxInt` for this value

Reply:
Done!